stripe-core/src/main/java/com/stripe/android/core/ApiKeyValidator.kt

```diff
@@ -18,6 +18,12 @@ class ApiKeyValidator {
                 "For more info, see https://stripe.com/docs/keys"
         }
 
+        require(!apiKey.startsWith("rk_")) {
```

Comment:
Can you add a test too?

Should be able to copy paste https://github.com/stripe/stripe-android/blob/d8460ee2f2c554df245cf7a0ca92d76be6195bca/stripe-core/src/test/java/com/stripe/android/core/ApiKeyValidatorTest.kt#L26